docs/source/setup/index.rst

```diff
@@ -1,16 +1,19 @@
 Getting started with SEPAL
 ==========================
 
-Introduction
+Set up an account and learn how to access the SEPAL interface
 ------------
 
-In this section of our documentation you will find the key requirements to use the platform. We'll start from :doc:`register` to the more advanced :doc:`filezilla`. use the left panel to navigate in this section of the documentation. 
-If you don't find what you need please let us know using the `GitHub issue tracker <https://github.com/openforis/sepal-doc/issues/new/choose>`_ of this very website.
+In this section of SEPAL documentation, you can learn about the key requirements needed to use the platform.
 
-We will be happy to receive contributions from people that use this documentation, so if you find a typo, don't hesitate to use the :btn:`<fas fa-pencil-alt> edit this page` button on the right side of every documentation page to help us improve the way we present the platform!
+This guide includes an Introduction to the SEPAL interface, as well as instructions that explain how you can :doc:`register`, Use Google Earth Engine (GEE) with SEPAL, :doc:`filezilla`, Use Norway’s International Climate & Forests Initiative (NICFI) - Planet Lab data, and Reset your password.
```

Comment:
```suggestion
This guide includes:
-   An Introduction to the SEPAL interface
-   Instructions that explain how you can :doc:`register`
-   Usage instructions for  Google Earth Engine (GEE) with SEPAL
-   Use of :doc:`filezilla` to download files
-   Use Norway’s International Climate & Forests Initiative (NICFI) - Planet Lab data
-   How to reset your password.
```

this is an example of how you can create a bullet point list. "-" and then 3 spaces, you can accept this suggestion and then modify the n text again if needed.